src/cclib/io/cjsonwriter.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, ccdata, terse=False, *args, **kwargs):
         Inputs:
           ccdata - An instance of ccData, parsed from a logfile.
         """
-
+        print ("TERSE:", terse)
```

Comment:
Should print `False` when called from the `testcjsonwriter.py` unittest. 
But prints `True`. This is why unittests did not fail for some files while ccwrite did (Molpro logs).